Libplanet/Action/ActionEvaluation.cs

```diff
@@ -138,6 +138,8 @@ int randomSeed
                     string msg;
                     if (!rehearsal)
                     {
+                        // FIXME: The below "exc" object seems never used; Hong Minhee believes
```

Comment:
`exc` seems used [for `new ActionEvaluation()`](https://github.com/planetarium/libplanet/pull/900/files#diff-37d0c58de3562a6f55cfb92ca89c8e9dR169-R174)

Reply:
No, this one is unused, because the same variable is always overwritten later:

https://github.com/planetarium/libplanet/blob/1ff9ffeeea623cf7375673521adfacd8690af74b/Libplanet/Action/ActionEvaluation.cs#L160-L162

Reply:
🆗